Solutions/0.2-Elliptic-Curve-Math-Solutions.ipynb

```diff
@@ -0,0 +1,328 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "import random\n",
+    "\n",
+    "sys.path.append('../')\n",
```

Comment:
Not needed.

More generally, how are you ensuring that the solution file is kept in sync with the base file when applying review markups?

Reply:
removed solution file